code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -107,7 +107,6 @@
 	var/obj/item/tank/jetpack/suit/jetpack = null
 	var/hardsuit_type
 
-
```

Comment:
This change seems unnecesary

Reply:
Left over from when I instead placed the Cybersun hardsuit in modular I believe.